build/azure-pipelines/sql-product-build.yml

```diff
@@ -41,15 +41,16 @@ jobs:
   steps:
   - template: win32/sql-product-build-win32.yml
 
-- job: Windows_Test
-  condition: and(succeeded(), eq(variables['VSCODE_BUILD_WIN32'], 'true'))
-  pool:
-    name: mssqltools
-  dependsOn:
-  - Linux
-  - Windows
-  steps:
-  - template: win32/sql-product-test-win32.yml
+# Temporarily disable Windows_Test job given build machine setup issues
```

Comment:
Might be worth trying a hosted windows agent while we investigate this further to see what kind of issues we hit on that since moving to something there would be ideal